docker/docker-compose.ci.yml

```diff
@@ -13,13 +13,22 @@ services:
     build:
       context: ../
       target: dev
+      cache_from: 
+        - unlockprotocol/unlock-dev
+      args:
+        BUILD_DIR: smart-contracts
   
   subgraph: # Subgraph unit testing
     image: subgraph
     platform: linux/amd64
     build:
       context: ../
+      target: subgraph
```

Comment:
?

Reply:
Should we remove the `target: dev` from below

Reply:
actually the subgraph target was removed